pkg/operator/ceph/csi/secrets.go

```diff
@@ -286,6 +286,9 @@ func CreateCSISecrets(context *clusterd.Context, clusterInfo *client.ClusterInfo
 	}
 
 	currentKeyCount := max(rbdProvKeyCount, rbdNodeKeyCount, cFSProvKeyCount, cFSNodeKeyCount)
+	if currentKeyCount <= 0 {
+		logger.Warningf("currentKeyCount is %d is less than or equal to 0, this could be due to an error in key generation.", currentKeyCount)
```

Comment:
Because we subtract one below, this should check `<= 1` 

```suggestion
	if currentKeyCount <= 1 {
		logger.Warningf("currentKeyCount %d is less than or equal to 1; this could be due to an error in key generation", currentKeyCount)
```

Reply:
Having only one key in the cluster and no prior key is still fine, lets discuss.